usaon_vta_survey/util/sankey.py

```diff
@@ -1,6 +1,9 @@
-from itertools import chain
-
-from usaon_vta_survey.models.tables import Response, ResponseDataProduct
+from usaon_vta_survey.models.tables import Response
+from usaon_vta_survey.util.full_sankey import (
+    _applications_sankey,
+    _data_products_sankey,
+    _societal_benefit_areas_sankey,
+)
```

Comment:
I feel like these functions should have stayed here, and `full_sankey` should import them. What do you think?

Reply:
That actually does make more sense!